go.mod

```diff
@@ -0,0 +1,39 @@
+module github.com/aws/amazon-cloudwatch-agent-test
+
+replace github.com/influxdata/telegraf => github.com/aws/telegraf v0.10.2-0.20220502160831-c20ebe67c5ef
+
+go 1.18
+
+require (
+	github.com/aws/aws-sdk-go v1.44.118
```

Comment:
The integration tests don't **need** the v1 SDK. Can you remove this and update imports as necessary? I think there are imports in the tests that use stuff like `aws.String()` but import from the v1 SDK when there are equivalent functions in v2. My e2e mem test PR included a change with that. Can follow that as a guide.

Reply:
oh this might have been before I removed the dependency to cwa main repo. will retry

Reply:
sure I can update